server/apps/publish/formatters/ninjs_formatter.py

```diff
@@ -35,8 +39,14 @@ def format(self, article, subscriber):
             for copy_property in self.direct_copy_properties:
                 if copy_property in article:
                     ninjs[copy_property] = article[copy_property]
+
+            if 'body_html' in article:
+                ninjs['body_xhtml'] = article['body_html']
```

Comment:
`body_html` is a perfectly fine field name, it's allowed by the NINJS standard. The "xhtml" suffix feels kind of archaic. :)